.tekton/integration-tests/deploy-operator.yaml

```diff
@@ -223,4 +223,49 @@ spec:
               - 30m
               - --namespace
               - "$(params.namespace)"
-              - "$(params.bundleImage)"
\ No newline at end of file
+              - "$(params.bundleImage)"
+    - name: e2e-tests
+      runAfter:
+        - deploy-operator
+      taskSpec:
+        volumes:
+          - name: credentials
+            emptyDir: {}
+        steps:
+          - name: get-kubeconfig
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/build-definitions.git
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml
+            params:
+              - name: eaasSpaceSecretRef
+                value: $(tasks.provision-eaas-space.results.secretRef)
+              - name: clusterName
+                value: "$(tasks.provision-cluster.results.clusterName)"
+              - name: credentials
+                value: credentials
+          - name: run-tests
+            env:
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+            volumeMounts:
+              - name: credentials
+                mountPath: /credentials
+            image: registry.redhat.io/rhel9/go-toolset:1.22.5
+            script: |
+              git clone https://github.com/openshift/instaslice-operator
+              curl -L -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.17/openshift-client-linux-amd64-rhel9.tar.gz \
```

Comment:
I thought @rphillips made an effort to remove a dependency on oc/kubelet in our e2e tests.

Reply:
In the test suite, yes.  But oc is needed to configure the OCP cluster.  Can't get rid of that dependency.